src/main/java/cn/nukkit/block/BlockLava.java

```diff
@@ -190,4 +194,9 @@ public void addVelocityToEntity(Entity entity, Vector3f vector) {
             super.addVelocityToEntity(entity, vector);
         }
     }
+
+
+    public static BlockFactory factory(Identifier stationaryId) {
```

Comment:
Do we need a factories method for this? The class is always going to have the same flowing and stationary Identifiers so it should be hardcoded in the constructor.